test/language/statements/for-in/identifier-let-allowed-as-lefthandside-expression-not-strict.js

```diff
@@ -1,11 +1,12 @@
 // Copyright (C) 2011 the V8 project authors. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 13.6.0.1
 description: >
-    for declaration:
     identifier "let" allowed as lefthandside expression
+esid: sec-iteration-statements
+info: |
+  for ( [ lookahead ∉ { let [ } ] LeftHandSideExpression [?Yield, ?Await] in 
+    Expression[+In, ?Yield, ? Await]) Statement[?Yield, ?Await, ?Return]
```

Comment:
Thanks for updating this, the information is extremely helpful for understanding the precise grammar being tested in the syntax below.

Reply:
lol, this syntax production is tricky to read, for a while I got confused expecting for a `let [` or `let` followed by a LHS. 

It's good to do some rubber ducking and explain to myself what is going on here.